test/filesystem.test

```diff
@@ -35,6 +35,13 @@ test_expect_success mkfs_f2fs,sload_f2fs,fsck_f2fs "f2fs" "
 	fsck.f2fs images/test.f2fs
 "
 
+exec_test_set_prereq mkfs.btrfs
+exec_test_set_prereq sload.btrfs
+exec_test_set_prereq fsck.btrfs
+test_expect_success mkfs_btrfs,fsck_btrfs "btrfs" "
+	run_genimage_root btrfs.config test.btrfs
```

Comment:
Run `btrfsck` here on the image. I don't think there is any useful output to compare, but at least we know it's a valid btrfs filesystem.

Reply:
Ok im go to add this.

Reply:
Fixed